### PR TITLE
fix(make): ensure latest pip before pip installing

### PR DESCRIPTION
**Pull request by joshuarli**
Ensure latest pip before pip install because really old pips will not have the pep517 disable flag.

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
Do we always want the latest version of pip? Should we pin it to a specific version?

**Comment by joshuarli** [comment]
> Do we always want the latest version of pip? Should we pin it to a specific version?

It doesn't matter too much, as long as we're on pip 19. Anyways, I just dug this related discussion up: https://github.com/getsentry/sentry-plugins/pull/504#discussion_r316722617 Then recently bumped+pinned it to 19.2.3 here: https://github.com/getsentry/sentry-plugins/pull/518

getsentry pins to `pip>=19,<20`, though it might be worthwhile to tighten it to 19.2.3. To set the example, I'll set it to 19.2.3 here.

File: Makefile [hunk 1]
```diff
@@ -7,16 +7,18 @@ ifneq "$(wildcard /usr/local/opt/openssl/lib)" ""
 	LDFLAGS += -L/usr/local/opt/openssl/lib
 endif
 
-PIP = LDFLAGS="$(LDFLAGS)" pip
-PIP_OPTS = --no-use-pep517 --disable-pip-version-check
-WEBPACK = NODE_ENV=production ./bin/yarn webpack
-YARN = ./bin/yarn
+PIP := LDFLAGS="$(LDFLAGS)" python -m pip
+# Note: this has to be synced with the pip version in .travis.yml.
```
Comment (billyvg) [inline Makefile]: Should this comment also go into `.travis.yml`

**Review by billyvg — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (joshuarli) [reply Makefile]: Yeah.

**Pull request merged**